cmd/kubeadm/app/cmd/init.go

```diff
@@ -387,6 +387,11 @@ func (i *Init) Run(out io.Writer) error {
 		return err
 	}
 
+	// Create/update RBAC rules that makes the nodes to rotate certificates and get their CSRs approved automatically
+	if err := nodebootstraptokenphase.AutoApproveNodeCertificateRotation(client, k8sVersion); err != nil {
```

Comment:
this should wait for #53338 to be approved, and then #53252 and #53338 should be picked together (`hack/cherry-pick.sh upstream/release-1.8 53252 53338`), since we don't want one merging without the other, right?

Reply:
No. #53338 is independent with certificate rotation issue. #53338 is just about 1.7.x -> 1.7.x upgrades with 1.8.0 kubeadm.